oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/util/ElasticIndexUtils.java

```diff
@@ -31,6 +31,31 @@ public class ElasticIndexUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(ElasticIndexUtils.class);
 
+    /**
+     * Convert a JCR property name to a Elasticsearch field name.
+     * Notice that "|" is not allowed in JCR names.
+     *
+     * "." is converted to "|dot|"
+     * "/" is converted to "||"
+     *
+     * @param propertyName the property name
+     * @return the field name
+     */
+    public static String fieldName(String propertyName) {
+        String fieldName = propertyName;
+        // 99% property names don't contain a slash or dot,
+        // so for performance reason use indexOf
+        int slashIndex = fieldName.indexOf('|');
```

Comment:
same here, should it not be `/`?

Reply:
Ah well, the field name `slashIndex` was wrong... but now changed.